src/OpenApi/src/PublicAPI.Shipped.txt

```diff
@@ -2,11 +2,11 @@
 Microsoft.AspNetCore.Builder.OpenApiEndpointConventionBuilderExtensions
 Microsoft.AspNetCore.Builder.OpenApiEndpointRouteBuilderExtensions
 Microsoft.AspNetCore.OpenApi.IOpenApiDocumentTransformer
-Microsoft.AspNetCore.OpenApi.IOpenApiDocumentTransformer.TransformAsync(Microsoft.OpenApi.Models.OpenApiDocument! document, Microsoft.AspNetCore.OpenApi.OpenApiDocumentTransformerContext! context, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task!
```

Comment:
Don't touch this file

Reply:
The codefixer for the PublicAPI analyzer applied this delta. Although I think it just reveals an underlying bug in the way the PublicAPI analyzer works? In this case, the same types for existing APIs have been moved to new namespaces. Maybe the delta here is to remove all the old APIs and then add the new ones?